app/(gcforms)/[locale]/(form filler)/id/[...props]/lib/rehydrateFormResponses.ts

```diff
@@ -74,9 +74,6 @@ function _rehydrateDynamicRowResponses(responses: [string, Response][]) {
 }
 
```

Comment:
Unrelated but it looks like we call JSON.parse() when checking every checkbox. Looks like some performance optimizations could be done here. (Future PR)